web/components/Modal.js

```diff
@@ -78,20 +78,24 @@ class ModalComponent extends Component {
         onClose: PropTypes.func.isRequired,
         children: PropTypes.any.isRequired
     }
+
```

Comment:
This is fine, but there will be no strict coding standards on whether or not to include a line break between component functions. It does improve readability which is good but this will not be enforced